### PR TITLE
Feature copy revision

### PR DESCRIPTION
**Pull request by anvaynk**

### CODE REVIEW

File: components/sections/JoinCommunity.tsx [hunk 1]
```diff
@@ -36,15 +36,14 @@ const JoinCommunity = () => {
             </Text>
             <VStack maxW={{ base: "md", md: "full" }} spacing="6" alignItems="start">
               <Text fontSize={{ base: "md", lg: "base" }} textAlign="left" color="brand.grey-40">
-                Shardeum is a global community. Anyone can join us in the journey to onboard
-                billions of people to Web 3. Don&apos;t know how to code? Or not sure where to
-                start? Don&apos;t worry. You can contribute as a content creator, community manager,
-                language translator, developer and you name it.
+                {
```
Comment (abhagsain) [inline components/sections/JoinCommunity.tsx:39]: I don't see any expressions so `{` can be removed.

File: components/sections/home/ReadWhitepaper.tsx [hunk 2]
```diff
@@ -51,23 +51,25 @@ function ReadWhitepaper() {
...
+                {/* Shardeum is a smart contract platform that scales{" "}
                 <Text as="span" color="brand.orange">
                   linearly
                 </Text>{" "}
                 and infinitely through{" "}
                 <Text as="span" color="brand.orange">
                   dynamic state sharding{" "}
-                </Text>{" "}
+                </Text>{" "} */}
+                The first linearly scalable smart contract platform that increases transactions per
```
Comment (abhagsain) [inline components/sections/home/ReadWhitepaper.tsx:62]: Highlighted text is missing? Was it intentional?
image

Main Website

image

**Review by abhagsain — CHANGES_REQUESTED** [review]

File: components/sections/JoinCommunity.tsx (hunk 1 above)
Comment (anvaynk) [reply components/sections/JoinCommunity.tsx:39]: Ok

File: components/sections/home/ReadWhitepaper.tsx (hunk 2 above)
Comment (anvaynk) [reply components/sections/home/ReadWhitepaper.tsx:62]: Fixed this

**Pull request merged**